src/util/__tests__/stripStyleAttribute.ts

```diff
@@ -0,0 +1,29 @@
+import stripStyleAttribute from '../stripStyleAttribute'
+
+it('only allowed properties', () => {
+  expect(stripStyleAttribute('color: red; font-weight: bold')).toBe('color: red;font-weight: bold;')
+})
```

Comment:
This test demonstrates that `stripStyleAttribute` does not strip `color` and `font-weight`, but it does not test that it strips other properties. Include a non-whitelisted property that is stripped for a complete test.

Reply:
There is a test like you said. The purpose of this test is to check the output for the allowed properties only.